### ISSUE TITLE
[Suggestion] Check for FPS Counter

### ISSUE THREAD

**Issue by LuanTeles** [aldostools/webMAN-MOD#1073]
Aldo, it's just a small adjustment. The FPS plugin combo shows the notification 'FPS Enabled' and 'FPS Disabled' even if the SPRX file is not present in the 'wm_res' folder. Can a check be added to not display the messages if the SPRX file is not there?

I'm just suggesting it because I use it via boot_plugins, so I always delete the one in 'wm_res' because I have an XMBML menu that controls the overlays. I realized that the notifications pop up even after deleting the SPRX file.

**Comment by aldostools** [comment]
I can add the check, but why isn't the plugin there? It's always installed. If the user or other apps are removing wMM files it's not my fault.

**Comment by LuanTeles** [comment]
I was deleting it because I use a modded one from the /plugins folder. Well, after posting my previous comment, I had an idea, and now I'm using a symlink so webMAN can load it, and the changes made via my XMBML menu reflect in webMAN's too (thanks for our previous talk), without being reset when a wmm update is installed.

So, I just realized that wmm doesn't have any check to display the notification, so I came up with the suggestion to check if the plugin exists first. It's not a big deal, I know.

**Comment by LuanTeles** [comment]
Ah, is there any way to create symlinks with webMAN Mod?

**Comment by aldostools** [comment]
Yes. /copy.ps3 will use symbolic links when you copy from hdd0 to hdd0.

If you want to force copy without symbolic links use //dev_hdd0 in the source or destination path.
